src/app/organization/common/organization.service.ts

```diff
@@ -19,8 +19,7 @@ export class OrganizationService {
   }
 
   getOrganization(id: number): Observable<Organization> {
-    const index = id - 1;
-    const url = organizationUrl + '/' + index;
+    const url = organizationUrl + '/' + id;
```

Comment:
could be string interpolation with backticks, i.e. `` `${organizationUrl}/${id}` ``